cypress/e2e/patient_spec/patient_files.cy.ts

```diff
@@ -0,0 +1,227 @@
+import { PatientEncounter } from "@/pageObject/Patients/PatientEncounter";
+import { PatientFiles } from "@/pageObject/Patients/PatientFiles";
+import { FacilityCreation } from "@/pageObject/facility/FacilityCreation";
+
+const facilityCreation = new FacilityCreation();
+const patientEncounter = new PatientEncounter();
+const patientFiles = new PatientFiles();
+
+describe("Patient Files", () => {
+  beforeEach(() => {
+    // cy.loginByApi("raj");
+    // cy.visit("/");
+    // facilityCreation.selectFacility("MEDICAL FACILITY");
+
+    cy.loginByApi("nurse");
```

Comment:
```suggestion
    cy.loginByApi("nurse");
```

- dont reuse exisitng id, create new one as we are connected to the local backend. So create new id and update the fixtures and use it here

Reply:
for testing in local i use new id (commented out),
can i push changes along with new id?